test/users.js

```diff
@@ -250,3 +250,51 @@ describe('/users/sessions POST', () => {
       });
   });
 });
+
+describe('/users GET', () => {
+  it('should list users', done => {
+    chai
+      .request(server)
+      .post('/users/sessions')
+      .send({
+        email: 'federico.casares@wolox.com.ar',
+        password: '12345678'
+      })
+      .then(response => {
+        chai
+          .request(server)
+          .get('/users')
+          .set('authorization', `Bearer ${response.body.token}`)
+          .then(res => {
+            expect(res).to.have.status(200);
+            expect(res).to.be.a.json;
+            expect(res.body).to.have.property('users');
+            dictum.chai(res, 'Users list endpoint');
+            done();
+          });
+      });
+  });
+
+  it('should fail if invalid token', done => {
+    chai
+      .request(server)
+      .get('/users')
+      .set('authorization', `Bearer 1234123`)
+      .end((err, res) => {
+        expect(err).not.to.be.null;
+        expect(res).to.have.status(500);
```

Comment:
If invalid or missing token the `express-jwt` package is responding 500?? I don't like that, 500 would mean that there is something wrong server-side, but clearly is something wrong with the client request, so it should be a 4xx.

Reply:
Fixed in #13